gensim/models/keyedvectors.py

```diff
@@ -765,9 +764,9 @@ def wmdistance(self, document1, document2):
         if diff1 > 0 or diff2 > 0:
             logger.info('Removed %d and %d OOV words from document 1 and 2 (respectively).', diff1, diff2)
 
-        if len(document1) == 0 or len(document2) == 0:
+        if not len(document1) or not len(document2):
```

Comment:
`not x:` is how you test for emptiness in Python.

Reply:
Ok, so are intending to use `is` keyword here? like `len(document1) is 0` ?

Reply:
not `is`, by `x` @piskvorky mean right side, like `not document1`, etc.

Reply:
Ok got you! Thanks @menshikh-iv